### DISCUSSION TITLE
Win+x Doesn't work

### DISCUSSION THREAD

**Discussion by Erich911** [valinet/ExplorerPatcher#3852] [General]
Hy
I use explorerpatcher ver 22621.3880.66.6 (last version )
All is fine except the shortcut : win+x 
If I uninstall explorerpatcher its works
I use start menu win10. Before the last microsoft update KB5044380 I think the shortcut did work.
Any suggestions to resolve it?

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3837

please close this.

**Comment by Erich911** [comment]
Thanks

**Comment by Erich911** [comment]
Thanks
